src/usr/share/gpgmailer/gpgmailer.py

```diff
@@ -49,6 +49,8 @@ def __init__(self, config, gpgkeyring, gpgkeyverifier):
         self.gpgmailbuilder = gpgmailbuilder.GpgMailBuilder(self.gpgkeyring,
             self.config['main_loop_duration'])
         self.mailsender = mailsender.MailSender(self.config)
+        self.valid_recipient_emails = []
+        self.valid_key_fingerprints = []
```

Comment:
I don't think these are used.

Reply:
Nope. Need to get these from the key verifier every time, passing in the time.